arch/x86_64/mcount-insn.c

```diff
@@ -476,7 +476,7 @@ static int check_instrumentable(struct mcount_disasm_engine *disasm, cs_insn *in
 		case CS_GRP_PRIVILEGE:
 		case CS_GRP_BRANCH_RELATIVE:
 			return INSTRUMENT_FAIL_NO_DETAIL;
-		default:
+		default:;
```

Comment:
Please don't mix two changes in a commit.  This change should be separate from the Github workflow.  Also you'd better have `break` statement in the following line.